code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -79,9 +79,13 @@
 
 		if(SSticker.current_state == GAME_STATE_PREGAME)
 			var/time_remaining = SSticker.GetTimeLeft()
-			if(time_remaining >= 0)
+			if(time_remaining > 0)
 				time_remaining /= 10
-			stat("Time To Start:", (time_remaining >= 0) ? "[round(time_remaining)]s" : "DELAYED")
+				stat("Time To Start:", "[round(time_remaining)]s")			
+			else if(time_remaining == -10)

```

Comment:
I don't recall -10 being a significant number in ticker?

Reply:
Since SSticker.GetTimeLeft() isn't being divided, -10 = delayed by admin. -20 = waiting for more players.